fabtests/common/shared.c

```diff
@@ -572,7 +572,7 @@ static int ft_init_oob(void)
 	int ret, op, err;
 	struct addrinfo *ai = NULL;
 
-	if (!(opts.options & FT_OPT_OOB_SYNC) || oob_sock != -1)
+	if ((!opts.oob_port && !(opts.options & FT_OPT_OOB_SYNC)) || oob_sock != -1)
```

Comment:
Why this change?

Reply:
The new option sets opts.oob_port w/o setting the FT_OPT_OOB_SYNC flag, but the oob channel still needs to be initialized.

Reply:
Maybe we should add Ft_OPT_OOB_ADDR_EXCH rather than relying on oob_port to convey this

Reply:
Yes, let me update the PR with that option.